### ISSUE TITLE
[BUG]: Checker having issues with ESM environments due to uuid usage

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#891]
### Project

a11y checker

### Description

https://ibm-studios.slack.com/archives/C036P1CTN/p1653658647868209
When using Jest 28.1.0, UUID is throwing errors due to ESM / CommonJS confusion. Need to figure out how to resolve usage in both environments.

### Steps to reproduce

See description

**Comment by areddon** [comment]
I resolved this by installing `uuid` as a dev dependency and using the jest module name mapper to point to my instance of uuid.

```
moduleNameMapper: {
   '^uuid$': '<rootDir>/node_modules/uuid/dist/index.js',
}
```

**Comment by shunguoy** [comment]
triage: this is caused by the node moving to ESM, we need to investigate and figure it out. The user suggestion is a temp workout. This should be in the next release when we are working on user-related work.

**Comment by tombrunet** [comment]
Working with our tests in the [4.0.0-rc.0](https://www.npmjs.com/package/accessibility-checker/v/4.0.0-rc.0) release. Please try and re-open if you still have issues with this release.
